src/components/list-with-actions/_list-with-actions.scss

```diff
@@ -4,22 +4,42 @@
 
 .hmrc-list-with-actions {
   @include govuk-font($size: 19);
+
+  @include govuk-media-query($from: desktop) {
+    .hmrc-summary-list__key {
+      width: 50% !important;
+    }
+
+    .hmrc-summary-list__actions {
+      width: 50% !important;
+    }
+  }
 }
 
-// Kept for backward compatibility with add-to-a-list
-.hmrc-list-with-actions--short {
+.hmrc-list-with-actions--long {
   @include govuk-media-query($from: desktop) {
+    .hmrc-summary-list__key {
+      width: 30% !important;
+    }
+
+    .hmrc-summary-list__actions {
+      width: 70% !important;
+    }
```

Comment:
@kyle-bowden should 21 to 27 move into .hmrc-list-with-actions--short instead of here (.hmrc-list-with-actions--long)? Or are we deliberately changing the meaning here between implementations, so it was short/long name before, now it's short/long actions? If we're changing meanings, I'd almost opt for a different modifier if that's the case, and then we could make it clear what the modifier does, for example "--short-actions" or even "--30/70" "--50/50" cc @timsb